src/App.js

```diff
@@ -4,21 +4,20 @@ import UserRegistration from './components/UserRegistration';
 import SignIn from './components/SignIn';
 import Home from './components/Home';
 import { Routes, Route, Navigate } from 'react-router-dom';
-import Logout from './components/Logout';
 import Navbar from './components/Navbar';
-import {
-  AuthStatus,
-  RequireAuth,
-  useAuth,
-} from '@akosasante/react-auth-context';
+import axios from 'axios';
+import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
 
 const URL = 'http://localhost:8000/api/v1/';
 
 function App() {
   const [message, setMessage] = useState('');
   const { status } = useAuth();
   const isLoggedIn = status === AuthStatus.LoggedIn;
-  // const navigate = useNavigate();
+  // create an axios instance
+  const instance = axios.create({
+    withCredentials: true,
+  });
```

Comment:
Instead of creating a new one, we should just import the one created in `src/apis/axiosClient.js`